playbooks/gcp/openshift-cluster/build_base_image.yml

```diff
@@ -80,11 +80,11 @@
   - role: openshift_repos
     vars:
       openshift_additional_repos: []
-  post_tasks:
   - name: Add custom repositories
```

Comment:
I couldn't test this though, is this the right syntax for pre_tasks?

Reply:
I'm not sure why this is fixing your issue - include_role is a task, not a role?

Reply:
To test this you need to run `make base-image FROM=centos-7 TO=some-made-up-image-name`

You then need to test the image with your own job by changing the base image to be `some-made-up-image-name`.  When you're done, you need to delete `some-made-up-image-name` and then we'll repush with the right defaults.